### DISCUSSION TITLE
Better game mode filtering for Multiplayer/Playlists lounge

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#13174] [Ideas]
**Describe the new feature:** 
i would suggest adding a game mode dropdown filter for multiplayer/playlists rather than using the selected game mode icon in the top toolbar.

![image](https://user-images.githubusercontent.com/43358824/103214810-cc8d0e00-4911-11eb-93bd-5d3021dbf88d.png)

i personally think this is a lot better and more intuitive since there is no real indicator that its already filtering the lobbys by the game mode. additionally a filter for "all" game modes would be nice to have.

![osu!_Z03ODYKz54](https://user-images.githubusercontent.com/43358824/103214604-24774500-4911-11eb-9e64-e42f8d1cbcb1.png)

**Comment by peppy** [comment]
If this is a consideration, then it's basically saying the whole game UX is broken, since those icons are deciding the filter applied to every screen.

**Comment by mouzedrift** [comment]
perhaps it is since there are additional buttons in some overlays like the ranking one.
![image](https://user-images.githubusercontent.com/43358824/103216067-630efe80-4915-11eb-8d7f-ccab35940037.png)

or in beatmap listing clicking on the top game mode icons doesn't change the beatmap listing filter.
![image](https://user-images.githubusercontent.com/43358824/103216168-b1bc9880-4915-11eb-9336-7935e1feb487.png)

the only place where you really need them is singleplayer and multiplayer if i didn't forgot something. but if someone would add the "Any" or "All" game mode category for the lounge then it would be for singleplayer only.

**Comment by peppy** [comment]
@arflyte something to read through / keep in mind.

**Comment by arflyte** [comment]
Noted.
